crates/bevy_ecs/src/query/access.rs

```diff
@@ -1,6 +1,6 @@
 use crate::storage::SparseSetIndex;
 use fixedbitset::FixedBitSet;
-use std::marker::PhantomData;
+use std::{collections::HashSet, marker::PhantomData};
```

Comment:
Unless there is a reason, could you use [`bevy_utils::HashSet`](https://docs.rs/bevy_utils/0.5.0/bevy_utils/type.HashSet.html) instead of the one from `std::collections`?

Reply:
Done